docs/network.md

```diff
@@ -0,0 +1,19 @@
+# 11. Code flow - routing a message
+This is the example of the message that is being sent between nodes (RawRoutedMessage) (https://github.com/near/nearcore/blob/fa8749dc60fe0de8e94c3046571731c622326e9f/chain/network-primitives/src/types.rs#L362)
+
+Each of these methods have a ‘target’ - that is either the account_id or peer_id or hash (which seems to be used only for route back..). If target is the account - it will be converted using ‘routing_table.account_owner’ to the peer.
```

Comment:
Yes hash is only used for routeback. Let's make that clear here.

Reply:
```suggestion
Each of these methods have a ‘target’ - that is either the account_id or peer_id or hash (which seems to be used only for route back..). If target is the account - it will be converted using `routing_table.account_owner` to the peer.
```